src/styles/_shared.scss

```diff
@@ -47,6 +47,10 @@ body {
     .cdc-react.checkbox-container .checkbox-content {
       display: none;
     }
+    .details-row.width-10 {
```

Comment:
does `.width-10` apply to anything? I don't see that in the markup but maybe one of the components creates it

Reply:
I think the `size="sm"` prop value adds that for the uswds styles.